scripts/add-yearn-vaults.js

```diff
@@ -42,14 +42,14 @@ async function main() {
   console.log(`hash ${transaction.hash}`);
   console.log(`Added new YearnV2 vault ${vault} for filter ${yearnV2Filter.address}`);
   configUpdate.yearn.v2.vaults.push(vault);
+  await configLoader.save(configUpdate);
```

Comment:
I believe it is sufficient to save the config only once at the end of the file.

Reply:
Yes but since there's a big delay between copying the SignHash, pasting it in the group chat, waiting for the signatures and copying them back, I felt more comfortable saving each change one after the other